pkg/resolve/resolve.go

```diff
@@ -145,7 +145,7 @@ func Resolve(ctx context.Context, cs *params.Run, logger *zap.SugaredLogger, pro
 		return []*tektonv1.PipelineRun{}, fmt.Errorf("could not find any PipelineRun in your .tekton/ directory")
 	}
 
-	if err := pipelineRunsWithSameName(types.PipelineRuns); err != nil {
+	if _, err := ResolvePipelineRunMetadata(types.PipelineRuns); err != nil {
```

Comment:
I kept this function in `Resolve`  in order to handle functionality for `tkn-pac resolve command`

Question is do we require to resolve `OriginalPRName` and duplication of name for command line ?

Reply:
i am not sure i understand your question.. what do you mean by name duplication